app/src/main/res/drawable/progress_soundcloud_horizontal_dark.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+	<item android:id="@android:id/background">
+		<shape>
+			<solid android:color="@color/dark_ripple_color" />
+		</shape>
+	</item>
+	<item android:id="@android:id/progress">
+		<clip>
+			<shape>
+				<solid android:color="@color/dark_soundcloud_primary_color" />
```

Comment:
I didn't test it, but you should be able to use `?attr/colorPrimary` to get the current primary colour and `?attr/colorControlHighlight` for the ripple colour.

Reply:
No, this will not work on pre-lollipop Android

https://stackoverflow.com/questions/41499168/unable-to-add-attribute-primary-color-in-android-drawable-xml

Reply:
Oh. That's good to know